exporters/otlp/profiles/src/main/java/io/opentelemetry/exporter/otlp/data/LinkData.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.exporter.otlp.data;
+
+import io.opentelemetry.api.internal.OtelEncodingUtils;
+import io.opentelemetry.api.trace.SpanId;
+import io.opentelemetry.api.trace.TraceId;
+import javax.annotation.concurrent.Immutable;
+
+/**
+ * A connection from a profile Sample to a trace Span.
+ *
+ * @see "pprofextended.proto::Link"
+ */
+@Immutable
+public interface LinkData {
+
+  /**
+   * Returns a unique identifier of a trace that this linked span is part of as 32 character
+   * lowercase hex String.
+   */
+  String getTraceId();
+
+  /** Returns the trace identifier as 16-byte array. */
+  default byte[] getTraceIdBytes() {
```

Comment:
Can we omit these and just have serializers know to reference `OtelEncodingutils.bytesFromBase16`?

Reply:
Sure. No reason they have to follow the same pattern as the existing SpanContext API does.